script.js

```diff
@@ -32,8 +32,18 @@ const strokeWeight = 1.5;
 const pointRadius = 8;
 const fillOpacity = 0.7;
 
+//setting the initial zoom settings
+if (document.querySelector("body").offsetWidth < 400){
```

Comment:
@goldfarb instead of the magic number `400` you could use the variable `MOBILE_BREAKPOINT` that already exists on line 7. I might also consider adding a variable, perhaps called `DESKTOP_BREAKPOINT` for the `1200` magic number. This way if these values need to be used elsewhere in the app they are already there.

You could also simplify this block by using the existing `IS_MOBILE` variable:

```js
if (IS_MOBILE) {
  z = 3;
} else {
  z = 4;
}
```

One last suggestion, it's generally a good practice to state a variable (even if it's not immediately set to a value), before conditionally set it. Otherwise it's being defined in the "global scope" and has greater potential to cause a bug. I would also give it a more descriptive name so that others know what it means before having to follow it in the code, e.g.

```js
let initialMapZoom = 4;
if (IS_MOBILE) {
  initialMapZoom = 3;
}
const map = L.map("map", mapOptions).setView([40.67,-97.23], initialMapZoom);
```

Reply:
good catch - I lost the `let z = ` when I copied the code out of my sandbox version

I'm happy to switch to using `IS_MOBILE`, I chose not to because the zoom levels don't line up well for showing the whole continental US